x-pack/solutions/security/packages/features/src/attack_discovery/kibana_features.ts

```diff
@@ -35,6 +43,10 @@ export const getAttackDiscoveryBaseKibanaFeature = (): BaseKibanaFeatureConfig =
         all: [],
         read: [],
       },
+      alerting: {
```

Comment:
Same alerts privileges as we have for the detection alerts https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/packages/features/src/security/v2_features/kibana_features.ts#L95